modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:cnv-vm-add-nic:
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-nic-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add NIC*.
+. In the *Add NIC* window, specify the *Name*, *Model*, *Network*, *Type*,
```

Comment:
It may be a question for QE, but are all those required?

Reply:
It appears so from the design @phoracek but best to let QE verify.
@alexxa can you verify these fields are all required in the UI, and if the UI steps are the same whether using a template or not? Thanks. Once I have that, we can merge and pick to 4.3.

Reply:
Confirmed.